src/status_im/contexts/chat/events.cljs

```diff
@@ -200,9 +200,10 @@
 (rf/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   {:events [:chat/navigate-to-chat]}
-  [{db :db :as cofx} chat-id animation]
+  [{db :db :as cofx} chat-id animation community-id]
   (rf/merge cofx
-            {:dispatch [(if animation :shell/navigate-to :navigate-to) :chat chat-id animation]}
+            {:dispatch [(if animation :shell/navigate-to [:navigate-to {:community-id community-id}]) 
+                        :chat chat-id animation]}
```

Comment:
I am confused here, how this works :thinking:

Reply:
oh probably it doesn't work, just gives illusion of working.

> we probably need to update :navigate-to-chat and just remove navigation part when chat is already opened

I think this change accidentally created this logic here. Removal of navigation part.

Reply:
In that case, we probably don't need to call `pop-to-root` or `:communities/navigate-to-community-overview` etc. It will just work without them.